stdlib/email/src/main/ballerina/src/email/listener_endpoint.bal

```diff
@@ -0,0 +1,196 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/java;
+import ballerina/lang.'object as lang;
+import ballerina/log;
+import ballerina/task;
+
+# Represents a service listener that monitors the email server location.
+public type Listener object {
+
+    *lang:Listener;
+
+    private handle EMPTY_JAVA_STRING = java:fromString("");
+    private ListenerConfig config = {};
+    private task:Scheduler? appointment = ();
+    private handle? serverConnector = ();
```

Comment:
Same here. Shall we make the type as `string?`

Reply:
This handle is not really related to a `string` type. Therefore, let's keep it as `handle?` type.